src/Compilers/CSharp/Portable/Syntax/SyntaxNormalizer.cs

```diff
@@ -206,7 +206,8 @@ private int LineBreaksAfter(SyntaxToken currentToken, SyntaxToken nextToken)
 
                 case SyntaxKind.CloseParenToken:
                     return (((currentToken.Parent is StatementSyntax) && nextToken.Parent != currentToken.Parent)
-                        || nextToken.Kind() == SyntaxKind.OpenBraceToken) ? 1 : 0;
+                        || nextToken.Kind() == SyntaxKind.OpenBraceToken
+                        || nextToken.Kind() == SyntaxKind.WhereKeyword) ? 1 : 0;
```

Comment:
>|| nextToken.Kind() == SyntaxKind.WhereKeyword [](start = 24, length = 46)

It would be good to add a comment explaining the scenario. Also, should we worry about over scenarios when ```where``` keyword is used, query expressions for example?

Reply:
`where` in a LINQ clause behaves the same. It's handled a few lines below (returns 1 when nextToken is `where`). I'll add a comment